src/Zoomable.js

```diff
@@ -86,7 +107,14 @@ class Zoomable {
 
       // remove the next sibling (the overlay)
       if (element.nextElementSibling) {
-        element.nextElementSibling.remove();
+        let overlay = element.nextElementSibling;
+
+        overlay.classList.add('fade-out');
```

Comment:
Since this whole thing is class-based, I think this is fine.

Wondering about switching things to be more event-driven (including firing events you could hook into)?